src/compiler/types.ts

```diff
@@ -3787,6 +3788,7 @@ export interface FileReference extends TextRange {
     preserve?: boolean;
 }
 
+/** @internal */
 export interface CheckJsDirective extends TextRange {

```

Comment:
This is used by tsutils :( https://github.com/search?q=%2F%5Cbts%5C.CheckJsDirective%2F+%28language%3ATypeScript+OR+language%3AJavaScript%29&type=code